api/pkg/client/mlflow/mlflow.go

```diff
@@ -0,0 +1,188 @@
+package mlflow
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type mlflowClient struct {
+	Client *http.Client
```

Comment:
Tiny nit but can we not have a `Client` in a `[...]Client`? 😅 Maybe have something like:
```go
type mlflowClient struct {
	api *http.Client
	Config Config
}
```

or maybe:

```go
type mlflowService struct {
	Client *http.Client
	Config Config
}
```